eahub/profiles/views.py

```diff
@@ -137,9 +138,15 @@ def edit_profile_community(request):
         form = EditProfileCommunityForm(request.POST, instance=request.user.profile)
         if form.is_valid():
             profile = form.save(commit=False)
+            profile.local_groups.clear()
             organisational_affiliations = request.POST.getlist('organisational_affiliations')
             profile.organisational_affiliations = organisational_affiliations
-            profile.save()
+            group_affiliations = request.POST.getlist('local_groups')
+            print(group_affiliations)
+            local_groups = LocalGroup.objects.filter(id__in=group_affiliations)
```

Comment:
I'm pretty sure this must be leaking database IDs. You should be able to solve this with [`to_field_name`](https://docs.djangoproject.com/en/2.1/ref/forms/fields/#django.forms.ModelMultipleChoiceField.to_field_name).

Reply:
I've just tried this:
```local_groups = forms.ModelMultipleChoiceField(queryset=LocalGroup.objects.all(), required=False, label="EA Groups", to_field_name="name")```
 but then it again shows with "LocalGroup" on the frontend.

The problem is that we're using django-crispy froms on the frontend, which doesn't give us direct control to the objects properties: The [example of to_field_name](https://docs.djangoproject.com/en/2.1/ref/forms/fields/#django.forms.ModelChoiceField.to_field_name) in the doc uses the field name, with django-crispy-forms all we do is putting in {{form.local_group}}, so we're not explicitly referencing the LocalGroup objects properties. I guess we could implement this without django-crispy forms, but what was then the point of adding it?